src/main/kotlin/no/nav/medlemskap/regler/v1/ReglerForLovvalg.kt

```diff
@@ -91,21 +96,23 @@ class ReglerForLovvalg(val datagrunnlag: Datagrunnlag) : Regler() {
         }
     }
 
-    private fun sjekkLandskode(): Resultat =
-            when {
-                sjekkBrukersPostadresseOgBostedsadresseLandskode() -> ja()
-                else -> nei("Bruker har enten ikke norsk bostedsadresse eller postadresse")
-            }
+    private fun sjekkLandkode(): Resultat {
+        val bostedsadresser = bostedsadresser.adresserSiste12Mnd(kontrollPeriodeForPersonhistorikk)
+        val postadresserLandkoder = postadresser.landkodeTilAdresseSiste12Mnd(kontrollPeriodeForPersonhistorikk)
+        val midlertidigadresserLandkoder = midlertidigAdresser.landkodeTilAdresseSiste12Mnd(kontrollPeriodeForPersonhistorikk)
+
+        return when {
+            bostedsadresser.erIkkeTom()
+                    && (postadresserLandkoder alleEr "NOR" || postadresserLandkoder.erTom())
+                    && (midlertidigadresserLandkoder alleEr "NOR" || midlertidigadresserLandkoder.erTom())-> ja()
+            else -> nei("Ikke alle adressene til bruker er norske, eller bruker mangler bostedsadresse")
```

Comment:
Har det en verdi å vite om det er det ene eller det andre man ryker på? Vi har ofte merket det i andre sammenhenger iallefall.. (Manglet personen bostedsadresse? Eller hadde hen en utenlands post/midl-adresse?)

Reply:
Ja, det har sikkert en verdi. Bare må tenke litt på hvordan jeg skal logge det.

Reply:
Har det en verdi å vite forskjellen både for oss og for klientene våre kanskje? Er det faktisk såpass at vi burde dele de to tingene opp i to separate regler?